src/smartcontracts/interaction.spec.ts

```diff
@@ -188,7 +188,7 @@ describe("test smart contract interactor", function () {
         assert.equal(transaction.getData().toString(), "getUltimateAnswer");
         assert.equal(
             transaction.getHash().toString(),
-            "60d0956a8902c1179dce92d91bd9670e31b9a9cd07c1d620edb7754a315b4818"
+            "3ff6309bfe87ecc4ab541db649d57187f3a012291570552ac003866f77212c19"
```

Comment:
Ideally, gas limit should stay the same (and the transaction hashes, as well).

Reply:
Fixed